app/models/state_file_w2.rb

```diff
@@ -91,7 +92,7 @@ def validate_tax_amts
         errors.add(:local_income_tax_amount, I18n.t("state_file.questions.w2.edit.wages_amt_error", wages_amount: w2.WagesAmt))
         errors.add(:state_income_tax_amount, I18n.t("state_file.questions.w2.edit.wages_amt_error", wages_amount: w2.WagesAmt))
       end
-      if state_wages_amount.present? && state_wages_amount > w2.WagesAmt
+      if state_wages_amount.present? && state_wages_amount > w2.WagesAmt && state_code != "NJ"
```

Comment:
[dust] is it worth making an enum or list out of state codes to not have to do string matching? I'm assuming they are all the official state abbreviations as used by USPS, which would make this pretty universal and low-risk... but my sense is it would be slightly more rubyish to do `state_code.is_nj?` etc?

Reply:
Agree that this seems like too high an effort/impact ratio

Reply:
works for me 👍